sequencer/src/context.rs

```diff
@@ -111,7 +124,14 @@ impl<N: network::Type, Ver: StaticVersionType + 'static> SequencerContext<N, Ver
             state_signer = state_signer.with_relay_server(url);
         }
 
-        Ok(Self::new(handle, persistence, node_id, state_signer))
+        Ok(Self::new(
+            handle,
+            persistence,
+            node_id,
+            state_signer,
+            Some(event_streamer),
```

Comment:
Why use `Option` if this is unconditionally `Some`?

Reply:
d1ba612c74d0131e4b06907bd3b5dd3b96aad345

Reply:
This is still `Option` and seems like it doesn't need to be